File: rev-com-manage/src/types/music/artist/artist.ts
import { ICategory } from "@/types/music/artist/category"
import { IType } from "@/types/music/artist/type"
import { IFileInfo } from "@/types/FileInfo"

interface IArtist extends IFileInfo {
  id: string
  name: string
  description: string
  avatarUrl: string
  area: ICategory
  type: IType
  createTime: string
  updateTime: string
}
export type { IArtist }

File: rev-com-server/src/main/java/com/codergb/bean/music/artist/Artist.java
package com.codergb.bean.music.artist;

import com.codergb.bean.FileInfo;
import lombok.*;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@EqualsAndHashCode(callSuper = true)
public class Artist extends FileInfo {
  private String id;
  private String name;
  private String description;
  private String avatarUrl;
  private ArtistCate area;
  private ArtistType type;
  private Date createTime;
  private Date updateTime;
}
